src/Symfony/Component/Serializer/Normalizer/AbstractNormalizer.php

```diff
@@ -205,22 +205,14 @@ protected function isCircularReference($object, &$context)
      *
      * @final since Symfony 4.2
```

Comment:
it seems we used to drop the `since ...` part of the `@final` flag https://github.com/symfony/symfony/pull/31672#pullrequestreview-243146149